src/sass/variables.scss

```diff
@@ -49,8 +49,8 @@ $mw-color-background-primary: $MW_COLOR_NEUTRAL_6;
 $mw-color-background-secondary: $MW_COLOR_NEUTRAL_5;
 $mw-color-border-primary: $MW_COLOR_NEUTRAL_4;
 $mw-color-border-secondary: $MW_COLOR_NEUTRAL_3;
-$mw-color-action: $MW_COLOR_BRAND_1;
-$mw-color-error: $MW_COLOR_ACCENT_2;
+$mw-color-action: $MW_COLOR_BRAND_3;
```

Comment:
If this is primarily used for text, then $MW_COLOR_BRAND_5 should be used. $MW_COLOR_BRAND_3 should be used for icons.